src/utils.js

```diff
@@ -0,0 +1,6 @@
+import { EMPTY } from './constants.js'
+
+export const isValidMove = (game, playerId, position) => {
+  const { board, playerTurn } = game
+  return board[position] === EMPTY && playerTurn === playerId
```

Comment:
What happens if we try to play a move, its our turn, the board position is empty, and the game is over?

Reply:
Still no validation for game over. Does the task include these functionality?